packages/@lwc/shared/src/api-version.ts

```diff
@@ -109,6 +111,10 @@ export const enum APIFeature {
      * If enabled, add support for complex class expressions in the template.
      */
     TEMPLATE_CLASS_NAME_OBJECT_BINDING,
```

Comment:
Just noticed that this isn't `ENABLE_` like the rest. Oh well, too late! 🫠

Reply:
It's not too late; we can change this...